docs/tidb_http_api.md

```diff
@@ -0,0 +1,52 @@
+#  TiDB HTTP API
+
+`TiDBIP` is the ip of tidb, `10080` is the default status port, and it can be changed in the tidb.toml when start. 
+
+> 1. Get present TiDB's status, about connections, version and git_hash
```

Comment:
-> Get the current status of TiDB, including the connections, version and git_hash